phpstan.neon.dist

```diff
@@ -1,3 +1,12 @@
+parameters:
+    level: 1
+
+    paths:
+        - %currentWorkingDirectory%/src
+        - %currentWorkingDirectory%/tests
+
+    tmpDir: %currentWorkingDirectory%/var/phpstan
```

Comment:
why change phpstan tmp dir? the default sys tmp seems to be good choose, no?

Reply:
I see, I'll revert it